en/apispec/openapi.json

```diff
@@ -3057,311 +2530,3053 @@
+                    "value": {
+                      "code": "MissingDateHeader",
+                      "message": "Authorized request must have a Date header.",
+                      "status": 0
+                    }
+                  },
+                  "RequestExpired": {
+                    "value": {
+                      "code": "RequestExpired",
+                      "message": "Request has expired. Timestamp date is xxx.",
+                      "status": 0
+                    }
+                  },
+                  "InvalidURI": {
+                    "value": {
+                      "code": "InvalidURI",
+                      "message": "Could not parse the specified URI.",
+                      "status": 0
+                    }
+                  },
+                  "InvalidStatus": {
+                    "value": {
+                      "code": "InvalidStatus",
+                      "message": "The value of status should be one of \"trial\", \"active\" and \"suspended\".",
+                      "status": 0
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "403": {
+            "description": "Forbidden",
+            "headers": {
+              "Qtl-Request-Id": {
+                "schema": {
+                  "type": "string"
+                },
+                "description": "A string automatically generated by the API server to identify each request."
+              }
+            },
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "properties": {
+                    "code": {
+                      "$ref": "#/components/schemas/response.code"
+                    },
+                    "message": {
+                      "$ref": "#/components/schemas/response.message"
+                    },
+                    "status": {
+                      "$ref": "#/components/schemas/response.status"
+                    }
+                  }
+                },
+                "examples": {
+                  "AccessDenied": {
+                    "value": {
+                      "code": "AccessDenied",
+                      "message": "A regular customer is not allowed to make this call.",
+                      "status": 0
+                    }
+                  }
+                }
+              }
+            }
+          }
+        },
+        "operationId": "get-serviceQuotas",
+        "description": "Returns a list of HDT service quotas that meet the conditions.\n\nNote:\nOnly super, partner, and reseller API accounts are allowed to make this call. Requests from regular API accounts shall receive a 403 error. Only service quotas for the child customers of the caller shall be returned. If the caller is a super customer, all service quotas shall be returned.  \n\nBoth “servicequotas” and “serviceQuotas” are accepted in the path.",
```

Comment:
In the CDN Pro documentation, we intentionally omitted any references to our "super" or "partner" customer types.
1) Some customers may be concerned about privacy 
2) The "partner" customer is only being used by Wangsu.
Therefore, there is no need to reveal to all customers.